airflow/scripts/git_scripts/aggregate_user_activty.py

```diff
@@ -0,0 +1,94 @@
+
+import requests
+from collections import defaultdict
+from datetime import datetime, timedelta
+
+
+token = 'put_token_here'   # add your token
+
+headers = {'Authorization': f'token {token}', 'Accept': 'application/vnd.github.v3+json'}
+
+# Define the date range
+since = (datetime.now() - timedelta(days=7)).isoformat()
+until = datetime.now().isoformat()
+
+
+def fetch_data(url, params=None):
```

Comment:
**relevant to all functions** 
use type hints for each function. (it will help you and other devs to understand the function input and outputs)
example: 
def fetch_data(url:str, params=None) -> json: 

For more information : https://docs.python.org/3/library/typing.html